### PR TITLE
Remove OpenTracing coverage for GraphQL

### PR DESCRIPTION
**Pull request by michalvavrik**
### Summary

GraphQL no longer supports OpenTracing and OpenTelemetry should be preferred https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.1#smallrye-graphql. We already have GraphQL with OpenTelemetry integration tests.

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [x] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: http/graphql/pom.xml [hunk 1]
```diff
@@ -15,50 +15,5 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-graphql</artifactId>
         </dependency>
-        <dependency>
```
Comment (rsvoboda) [inline http/graphql/pom.xml:18]: Shouldn't be the coverage converted to opentelemetry instead?

File: http/graphql/pom.xml [hunk 2]
```diff
@@ -15,50 +15,5 @@
...
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-smallrye-opentracing</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.quarkus.qe</groupId>
-            <artifactId>quarkus-test-service-jaeger</artifactId>
-            <scope>test</scope>
-        </dependency>
     </dependencies>
-    <profiles>
```
Comment (rsvoboda) [inline http/graphql/pom.xml:28]: Why are the profiles removed?

Comment (michalvavrik) [reply http/graphql/pom.xml:28]: We can run tests on Windows as now there is no container. We don't need extension dependency here as it is not used.

File: http/graphql/pom.xml (hunk 1 above)
Comment (michalvavrik) [reply http/graphql/pom.xml:18]: I trust https://github.com/quarkus-qe/quarkus-test-suite/tree/main/http/graphql-telemetry if we need to extend coverage, it should be separate PR.

**Review by rsvoboda — APPROVED** [review]

**Comment by rsvoboda** [comment]
Too quick

**Comment by rsvoboda** [comment]
one profile should stay, that `deploy-to-openshift-using-extension`

**Comment by michalvavrik** [comment]
> one profile should stay, that deploy-to-openshift-using-extension

why?

**Comment by rsvoboda** [comment]
https://github.com/quarkus-qe/quarkus-test-framework/blob/main/quarkus-test-openshift/src/main/java/io/quarkus/test/services/quarkus/ExtensionOpenShiftQuarkusApplicationManagedResource.java#LL35C33-L35C56

**Comment by rsvoboda** [comment]
What do you mean? I see OpenShift scenarios in https://github.com/quarkus-qe/quarkus-test-suite/tree/main/http/graphql/src/test/java/io/quarkus/ts/http/graphql

**Comment by michalvavrik** [comment]
Basically it is a way to automatically activate profile when we use `io.quarkus.test.scenarios.OpenShiftDeploymentStrategy#UsingOpenShiftExtension` or the other ex. But we use `build` instead.

**Comment by rsvoboda** [comment]
OK, `OpenShiftDeploymentStrategy.Build` is the default, profile definition relates to `OpenShiftDeploymentStrategy.UsingOpenShiftExtension`

**Pull request merged**